### PR TITLE
chore(test/apply_fun): quiet test file

### PR DESCRIPTION
**Pull request by adomani**
This is part of the process of quieting test files.  Since I was not really able to automate this change, I am PRing it separately.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
This doesn't seem like a great approach to me; if any of the tactics being tested try to pull hypotheses from their context, these tests will break.

Can we just silence `sorry` in tests somehow? Or introduce a `test_sorry` axiom that is silent?

**Comment by adomani** [comment]
I am happy to introduce a `test_sorry` axiom.  I had used systematically `#guard_msgs` in #6868, but someone, I think Mario, did not like it.

**Comment by eric-wieser** [comment]
Can you link to where Mario objected?

**Comment by eric-wieser** [comment]
Let's wait for @digama0 to agree on that strategy before throwing away more work!

File: test/apply_fun.lean [hunk 1]
```diff
@@ -250,10 +250,10 @@ example (x : ℕ) : x = x := by
   rfl
 
 -- Check that locals are elaborated properly in apply_fun
-example : 1 = 1 := by
+example (h : ∀ {α β}, ∀ f : α → β, Function.Injective f): 1 = 1 := by
```
Comment (digama0) [inline test/apply_fun.lean:253]:
```suggestion
example (h : ∀ {α β}, ∀ f : α → β, Function.Injective f) : 1 = 1 := by
```

**Comment by adomani** [comment]
`apply_fun` was quieted in #6868.

**Pull request closed without merging**